testing/elf_loader.h

```diff
@@ -15,6 +15,8 @@ namespace flutter {
 namespace testing {
 
 inline constexpr const char* kAOTAppELFFileName = "app_elf_snapshot.so";
+inline constexpr const char* kAOTAppELFSplitFileName =
```

Comment:
Not sure how this name was generated. I see the fixture generator specifying the app_elf_snapshot base name but unclear about the `so-2.part` bit. Can you add a comment here for readers on how the tooling generates this name? Or, just be explicit about the name if the capability to do so is already present.

Reply:
Yeah, sure. This naming is just what gen_snapshot defaults to. The `2` indicates the id for the loading unit is 2 (base module is 1).